### DISCUSSION TITLE
Does sftpgo support multiple S3 buckets per SFTP host?

### DISCUSSION THREAD

**Discussion by jamondigimon** [drakkan/sftpgo#1167] [Q&A]
According to the [tutorial doc postgresql-s3.md](https://github.com/drakkan/sftpgo/blob/main/docs/howto/postgresql-s3.md), it is feasible to have a single s3 bucket corresponding to a SFTP host, and the setup steps are given in the docs.

Is it possible to have multiple s3 buckets tied to one SFTP host? Any suggestions & pointers are warmly welcomed!

**Comment by drakkan** [comment]
Yes it supported, you can have a different bucket for each user and even multiple buckets for a single user.

**Reply by jamondigimon** [reply]
Cool thanks! Does this require special setup or can be done via the UI?

**Reply by drakkan** [reply]
> Cool thanks! Does this require special setup or can be done via the UI?

the UI is enough
